test/e2e_node/gubernator.sh

```diff
@@ -63,6 +63,10 @@ if ! gsutil ls gs:// | grep -q "gs://${bucket_name}/"; then
 else
   V=2 kube::log::status "Bucket already exists"
 fi
+
+# Make all files in the bucket publicly readable
+gsutil acl ch -u AllUsers:R gs://${bucket_name}
```

Comment:
Move this into the if block where you create the bucket. If it already exists, we don't want to mess with the permissions on it.